File: Example/SHViewPager/SHContentViewController.m
//
//  SHContentViewController.m
//  SHViewPager
//
//  Created by shabib hossain on 8/28/16.
//  Copyright © 2016 Code with Shabib. All rights reserved.
//

#import "SHContentViewController.h"

@interface SHContentViewController ()

@property (nonatomic, weak) IBOutlet UILabel *selectionLabel;

@end

@implementation SHContentViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view from its nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (void)viewWillAppear:(BOOL)animated {
    _selectionLabel.text = _selectionIndicatorString;
}

@end

File: Example/SHViewPager/SHContentViewController.swift
//
//  SHContentViewController.swift
//  SHViewPager
//
//  Created by Shabib Hossain on 8/29/16.
//  Copyright © 2016 shabib87. All rights reserved.
//

import UIKit

class SHContentViewController: UIViewController {

    var selectionIndicatorString: String!
    @IBOutlet private var indicatorLabel: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        indicatorLabel.text = selectionIndicatorString
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

}
